spec/travel_advice_publisher_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+require "plek"
+require "faker"
+require "date"
+
+describe "travel advice publisher", type: :feature, travel_advice_publisher: true do
+  feature "Creating and deleting new editions" do
+    let(:title) { title_timestamp }
+    let(:summary) { Faker::Lorem.paragraph }
+    let(:part_title) { Faker::Book.title }
+    let(:part_body) { Faker::Lorem.sentence }
+    let(:country) { randomly_select_country }
```

Comment:
If we're to use a random country here we open up the possibility that tests fail for a particular country and not others - thus making the tests non-deterministic (for an example we've had problems with this on Publishing API failing on randomly generated schemas).

So I want to check we're doing this for the right reason and that the test output makes it clear which country the test has failed for - otherwise if we get a country that tests fail for we'll just run them again and think the suite is flaky.